.github/workflows/run-tests.yml

```diff
@@ -145,3 +156,35 @@ jobs:
           CLICKHOUSE_CLOUD_PASSWORD: ${{ secrets.INTEGRATIONS_TEAM_TESTS_CLOUD_PASSWORD }}
         run: |
           npm run test:integration:cloud
+
+  upload-coverage-and-badge:
+    if: github.ref == 'refs/heads/main'
+    needs:
+      - integration-tests-local-single-node
+      - integration-tests-local-cluster
+      - integration-tests-cloud
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Setup NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Download coverage report
+        uses: actions/download-artifact@v3
+        with:
+          name: coverage
+          path: coverage
+      - name: Install packages
+        run: npm i -G make-coverage-badge
+      - name: Generate badge
+        run: npx make-coverage-badge
```

Comment:
Does `npx` require `install` step in advance?

Reply:
ah, unfortunately it prompts and there seems to be no `--yes` option:

```
➜  ~ npx make-coverage-badge
Need to install the following packages:
  make-coverage-badge@1.2.0
Ok to proceed? (y)
```